Update managed-application-publishing.md

jrudley opened the pull request:
az cli is case sensitive. Owner needs to have a capital O else it returns no results
az role definition list --name owner <- no results returned
az role definition list --name Owner <-correct way

azurecla commented:
Hi __@sachouks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sachouks). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jrudley committed:
Update managed-application-publishing.md

az cli is case sensitive. Owner needs to have a capital O else it returns no results
az role definition list --name owner <- no results returned
az role definition list --name Owner <-correct way
